src/main/java/dev/tssvett/schedule_bot/bot/formatter/ScheduleStringFormatter.java

```diff
@@ -1,108 +1,106 @@
-
-        for (LessonInfoDto lesson : dayLessons) {
-            sb.append(formatLesson(lesson));
-        }
+    private String emptyEducationalDayToString(String weekDayName) {
+        StringBuilder sb = new StringBuilder();
+        String currentDate = LocalDate.now().format(DateTimeFormatter.ofPattern("dd.MM.yyyy"));
+        sb.append(String.format("""
+                %s %s (%s):
+                Зачилься, пар нет :)
+                """, MessageTextConstantsUtils.DAY_HEADER, capitalizeFirstLetter(weekDayName), currentDate));
+        sb.append("\n");
 
-        sb.append("\n"); // Добавляем пустую строку для разделения дней
+        return sb.toString();
     }
 
-    private String formatLesson(LessonInfoDto lesson) {
-        String emoji = getEmojiForLesson(lesson);
+    private String dayHeaderToString(String day, String lessonDate) {
+        return String.format("""
+                %s %s (%s):
+                """, MessageTextConstantsUtils.DAY_HEADER, capitalizeFirstLetter(day), lessonDate);
+    }
 
-        return String.format(
-                "%s %s\n" + // Смайлик и название пары
-                        "Тип: %s\n" +
-                        "Место: %s\n" +
-                        "Время: %s\n\n",
-                emoji,
-                capitalizeFirstLetter(lesson.name()),
+    private String lessonToString(LessonInfoDto lesson) {
+        return String.format("""
+                        %s %s
+                        Тип: %s
+                        Место: %s
+                        Время: %s
+                                    
+                        """, getEmojiForLesson(lesson), capitalizeFirstLetter(lesson.name()),
                 lesson.type(),
                 lesson.place(),
-                lesson.time()
-        );
+                lesson.time());
     }
 
     private String getEmojiForLesson(LessonInfoDto lesson) {
-        if (lesson.place().equalsIgnoreCase("online")) {
-            return ONLINE_EMOJI;
-        } else if (lesson.type().equalsIgnoreCase("лабораторная")) {
-            return LAB_EMOJI;
-        } else if (lesson.type().equalsIgnoreCase("лекция")) {
-            return LECTURE_EMOJI;
-        } else if (lesson.name().equalsIgnoreCase("Военная кафедра")) {
-            return MILITARY_EMOJI;
-        } else {
-            return DEFAULT_EMOJI;
+        String place = lesson.place().toLowerCase();
+        String type = lesson.type().toLowerCase();
+        String name = lesson.name().toLowerCase();
+
+        if ("online".equals(place)) {
+            return MessageTextConstantsUtils.ONLINE_EMOJI;
         }
+
+        return switch (type) {
+            case "лабораторная" -> MessageTextConstantsUtils.LAB_EMOJI;
+            case "лекция" -> MessageTextConstantsUtils.LECTURE_EMOJI;
+            default -> "военная кафедра".equals(name) ? MessageTextConstantsUtils.MILITARY_EMOJI
```

Comment:
Может так лучше?
```suggestion
            default -> name.equals("военная кафедра") ? MessageTextConstantsUtils.MILITARY_EMOJI
```
А то странно немного выглядит такой синтаксис

Reply:
нет. твой вариант не лучше

если name null  - будет NPE